runtime/hydradx/src/governance.rs

```diff
@@ -55,24 +117,41 @@ impl pallet_treasury::Config for Runtime {
 	type Burn = Burn;
 	type PalletId = TreasuryPalletId;
 	type BurnDestination = ();
-	type WeightInfo = weights::treasury::HydraWeight<Runtime>;
+	type WeightInfo = weights::pallet_treasury::HydraWeight<Runtime>;
 	type SpendFunds = ();
 	type MaxApprovals = MaxApprovals;
-	type SpendOrigin = NeverEnsureOrigin<Balance>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type SpendOrigin =
+		frame_system::EnsureWithSuccess<EnsureRoot<AccountId>, AccountId, crate::benches::BenchmarkMaxBalance>;
+	type AssetKind = ();
+	type Beneficiary = AccountId;
+	type BeneficiaryLookup = IdentityLookup<AccountId>;
+	type Paymaster = PayFromTreasuryAccount; // TODO: check what this means
+	type BalanceConverter = UnityAssetBalanceConversion; //TODO: check this
+	type PayoutPeriod = TreasuryPayoutPeriod;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = (); //TODO: implement helper!
```

Comment:
I believe the default implementation should be ok. Wdyt @enthusiastmartin ?

Reply:
i believe it is not needed. it only needed to create special asset and account if needed. and benchmarks seem to work without it.
this would be needed if we use `Paymaster` and different asset and account.